### DISCUSSION TITLE
Login Issues when TTV LOL PRO is Enabled

### DISCUSSION THREAD

**Discussion by PHY1643** [younesaassila/ttv-lol-pro#212] [Q&A] [answered]
I'm not convinced this is a bug with the add-on, so I thought I'd post here first. I'm running addon version 2.1.0 for Firefox (114.0.1) on Linux (ubuntu variant). I have the add-on configured with the defaults (nothing customized).

Lately, I've noticed that if the add-on is enabled when I attempt to login to Twitch, I receive a rather vague error on the login screen ("Twich encountered an error" or some such nonsense.) It seems that login is successful only when the add-on is disabled. I've tested this a handful of times over the past few days, mixing and matching with clearing cache/cookies/fresh profile/etc., and the results are always the same. In any event, once I'm logged in I can re-enable the add-on, reload a stream, and everything works exactly as intended.

Is anyone else experiencing this?

**Comment by klydeinside** [comment]
Yes. It's a VPN issue. It seems they might be cracking down on them. Or maybe they're matching your language to the VPN you're using and they know it's not matching so they block it. I'm not sure but I've had it happen and to get in I had to disable any VPNs I was using.

**Reply by PHY1643** [reply]
I'm willing to entertain the idea that a VPN is related, since I use a VPN service 100% of the time (via router hardware). _However_, I don't think I fully buy that the VPN is the _cause_. The reason I say that is because when the TTVLOLPRO add-on is disabled, I can log in to Twitch with no issues, VPN or no. But I suppose it could be possible that Twitch is "cracking down" on those who are using _both_ a VPN service and the TTVLOLPRO add-on... (That could very well be the case -- I honestly cannot say.) In any event, I think I was correct to assume this is _not_ a bug. Thanks.

**Reply by zGato** [reply, marked as answer]
Sorry for the late response. You have your response [here](https://github.com/younesaassila/ttv-lol-pro/issues/198)

I'd recommend you to use that VPN on your PC directly and proxy to Russia or some other ad-free country for the best experience (I suppose your router could forward to DIRECT without VPN via a Rule? Or via domain?)

And, if you have a server on your home (or you're a homelab guy and want to test things) check out this project https://github.com/qdm12/gluetun
You can expose your VPN as a simple HTTP proxy and so forward only X requests thru that VPN

**Reply by PHY1643** [reply]
Thanks!
